clang/tools/libclang/CDependencies.cpp

```diff
@@ -53,8 +53,11 @@ void clang_experimental_ModuleDependencySet_dispose(
 
 CXDependencyScannerService
 clang_experimental_DependencyScannerService_create_v0(CXDependencyMode Format) {
+  // FIXME: Pass default CASOpts and nullptr as CachingOnDiskFileSystem now.
+  CASOptions CASOpts;
+  IntrusiveRefCntPtr<llvm::cas::CachingOnDiskFileSystem> FS;
   return wrap(new DependencyScanningService(
-      ScanningMode::DependencyDirectivesScan, unwrap(Format),
+      ScanningMode::DependencyDirectivesScan, unwrap(Format), CASOpts, FS,
```

Comment:
The fix in this file is just get things compiling. I don't think we need them anymore